main.js

```diff
@@ -220,6 +227,28 @@ define(function (require, exports, module) {
         }
     }
     
+    function _applyColors() {
+        // Slight delay so Brackets can apply the dark class
+        setTimeout(function() {
+            var userColors = _preferences.get("colors"),
+                toApply    = $("body").hasClass("dark") ? userColors.dark : userColors.light,
+                css        = ".CodeMirror .cm-dk-whitespace-space:before," +
+                             ".CodeMirror .cm-dk-whitespace-tab:before," +
+                             ".CodeMirror .cm-dk-whitespace-leading-space:before,"  +
+                             ".CodeMirror .cm-dk-whitespace-leading-tab:before" +
+                             "{background-color:" + toApply[0] + ";} " +
+                             ".CodeMirror .cm-dk-whitespace-trailing-space:before, "  +
+                             ".CodeMirror .cm-dk-whitespace-trailing-tab:before" +
+                             "{background-color:" + toApply[1] + ";} ";
+
+            if (!_styleInline) {
+                _styleInline = ExtensionUtils.addEmbeddedStyleSheet(css);
+            } else {
+                $(_styleInline).text(css);
+            }
+        }, 100);
```

Comment:
I'm not at all liking this timeout, but it was the only way I could it working (we have to wait for Brackets to apply the `dark` class before doing anything. Thoughts?